edit-post/components/sidebar/post-author/style.scss

```diff
@@ -1,3 +1,11 @@
 .editor-post-author__select {
 	margin: -5px 0;
+
+	// Set the width of the author select box in IE11 to prevent it overflowing
+	// outside of the container because of IE11 flexbox bugs.
```

Comment:
Totally fair. I'll add a follow-up PR with some more context after I'm back from squash. IE11 is full of flex box bugs, really, it can be hard to keep track of which is which sometimes. 😓

Reply:
Yeah, my go-to has been https://github.com/philipwalton/flexbugs . Not sure if it's included somewhere in there.